Dockerfile

```diff
@@ -33,9 +21,36 @@ COPY pkg/ pkg/
 COPY patches/ patches/
 COPY templates/ templates/
 
-# Build
+###### Dev stage - start ######
+
+FROM common AS dev
+
 USER root
 
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o model-registry
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as dev-build
+
+WORKDIR /
+COPY --from=dev /workspace/model-registry .
+USER 65532:65532
+
+ENTRYPOINT ["/model-registry"]
```

Comment:
So when the `--target=dev-build` is active, it will stop completing this section, to my understanding.

(and when no specific `--target` is provided, eg building the full image, it progresses)

Reply:
@tarilabs It's actually better that! container tools, when no target is specified, take the latest (from the bottom) by default and build a dependency graph from there, since `dev-build` is not required from the latest (unnamed target), it gets skipped 😁 If you specify `dev-build` as the target instead, it starts from there and builds its dependency graph, skipping the rest of the container file

Reply:
you mean like taking the "branch" here : 

```mermaid
flowchart TD
    common[registry.access.redhat.com/ubi8/go-toolset:1.23 AS common]
    dev[common AS dev]
    dev-build[registry.access.redhat.com/ubi8/ubi-minimal:latest as dev-build]
    builder[common as builder]
    default[registry.access.redhat.com/ubi8/ubi-minimal:latest]
    common --> dev
    dev --> dev-build
    common --> builder
    builder --> default
```